### DISCUSSION TITLE
VSCode Auto-Import doesn't work

### DISCUSSION THREAD

**Discussion by kaikaibenkai** [nuxt/nuxt#19517] [Questions]
I have just upgraded Nuxt 3.1 to 3.2...

![image](https://user-images.githubusercontent.com/99722991/223578597-a0143959-e311-443d-a685-e815d7ea3ec3.png)

How to make it work properly? (starting server can be successful)

---

Demo: https://github.com/kaikaibenkai/nuxt-demo-19517

**Comment by AloisSeckar** [comment]
I'd suggest you start with deleting `node_modules` and `package-lock.json` and then re-run `npm install` and `npm run dev` 
(or respective commands, if you use yarn or pnpm)

**Reply by kaikaibenkai** [reply]
I tried some times, but didn't solve the problem.
Here's the [demo](https://github.com/kaikaibenkai/nuxt-demo-19517). Can you help me?

**Reply by AloisSeckar** [reply]
I only see Nuxt 3 minimal starter in the given repo. No files with `defineNuxtPlugin`
